pandas/tests/extension/test_numpy.py

```diff
@@ -236,6 +236,7 @@ def test_getitem_scalar(self, data):
 
 
 class TestGroupby(BaseNumPyTests, base.BaseGroupbyTests):
+    @skip_nested
```

Comment:
Here (and in line 407 below) I added `@skip_nested` in imitation of the surrounding classes.  Is that the right thing to do?  Do I need to do `@pytest.mark.parametrize`?

Reply:
These changes caused tests to fail, so I removed the annotations and instead added comments to suppress the Pylint warnings.